recipes/sleap-io/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sleap-io" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sleap_io-{{ version }}.tar.gz
+  sha256: 82fb6ca03975d833fc83b5174a04ab63dff652fa9d660767762b721f4c0f3dd3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
```

Comment:
can we increase this to python 3.8 or even 3.9? honestly its REALLY hard to maintain working packages accross all version of python. 

I would venture to say if your users are starting off today, they likely aren't using python 3.7, at least not on conda-forge.

Reply:
Is there any downside with being more permissive in the run section? It technically _does_ work in 3.7 and we work in some environments where 3.7 is still running.

If no one uses it, then no harm, no foul?

Reply:
yeah its probably ok. just was trying to reduce variables.